platforms/vercel/now.json

```diff
@@ -1,6 +1,6 @@
 {
   "version": 2,
-  "name": "e2e-platforms-zeit-now",
+  "name": "e2e-platforms-vercel",
   "env": {
     "ZEIT_NOW_PG_URL": "@zeit_now_pg_url"
```

Comment:
Nit: `    "ZEIT_NOW_PG_URL": "@zeit_now_pg_url"`

Reply:
Thanks, noted, will piggyback this when I touch E2E next :)